app/templates/admin/permissions/system-roles.hbs

```diff
@@ -46,10 +46,10 @@
       </td>
       <td>
         <div class="ui horizontal compact basic buttons">
-          {{#ui-popup content=(t 'Edit') class='ui icon button' position='left center'}}
+          {{#ui-popup content=(t 'Edit') class='ui icon button'}}
             <i class="edit icon"></i>
           {{/ui-popup}}
-          {{#ui-popup content=(t 'Delete') class='ui icon button' position='left center'}}
```

Comment:
@CosmicCoder96 `left center` should have been done only for vertical menu. As here the menu is horizontal `left center` hides the other option in same menu. That's why I removed it here
![screenshot 4](https://user-images.githubusercontent.com/12716067/27389468-1723d1ac-56bc-11e7-9869-ff1e95993567.png)